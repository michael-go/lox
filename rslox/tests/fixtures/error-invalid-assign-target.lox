var breakfast;
breakfast + "foo" = "bar";