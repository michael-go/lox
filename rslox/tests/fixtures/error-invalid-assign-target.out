# exit code: 65
# stdout:

# stderr:
[line 2] Error at '=': Invalid assignment target.

